src/main/webapp/resources/js/pages/analysis/components/AnalysisDelete.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState, useContext } from "react";
+import { Button, Checkbox, Alert, Popconfirm, Row, Typography } from "antd";
+import { AnalysisContext } from "../../../contexts/AnalysisContext";
+import { showNotification } from "../../../modules/notifications";
+import { getI18N } from "../../../utilities/i18n-utilties";
+
+import { deleteAnalysis } from "../../../apis/analysis/analysis";
+const { Title } = Typography;
+
```

Comment:
Comments

Reply:
Fixed in f8844707a5f60a885ba87a349edb62b076eaba15